test/sequential/rolldown.forceUpdate.test.ts

```diff
@@ -0,0 +1,230 @@
+/*
+ *
+ * @group sequencerCancellation
+ */
+
+import {
+  ChainName,
+  SequencerStaking,
+} from "../../utils/rollDown/SequencerStaking";
+import {
+  L2Update,
+  Rolldown,
+  createAnUpdate,
+  createAnUpdateAndCancelIt,
+  leaveSequencing,
+} from "../../utils/rollDown/Rolldown";
+import { getApi, initApi } from "../../utils/api";
+import { setupApi, setupUsers } from "../../utils/setup";
+import { Sudo } from "../../utils/sudo";
+import { BN_MILLION, signTx } from "gasp-sdk";
+import {
+  ExtrinsicResult,
+  findEventData,
+  waitNewBlock,
+  waitSudoOperationSuccess,
+} from "../../utils/eventListeners";
+import { isBadOriginError } from "../../utils/utils";
+import { getEventResultFromMangataTx } from "../../utils/txHandler";
+import { AssetWallet, User } from "../../utils/User";
+import { Assets } from "../../utils/Assets";
+import { GASP_ASSET_ID } from "../../utils/Constants";
+
+let api: any;
+let testUser: User;
+let testUserAddress: string;
+let chain: any;
+
+async function setupASequencer(user: User, chain: ChainName = "Ethereum") {
+  const extrinsic = await SequencerStaking.provideSequencerStaking(
+    (await SequencerStaking.minimalStakeAmount()).addn(1000),
+    chain,
+  );
+  await Sudo.batchAsSudoFinalized(
+    Assets.mintNative(user),
+    Sudo.sudoAs(user, extrinsic),
+  );
+}
```

Comment:
Is not elsewhere a function that does exactly that?

Reply:
No, the another function doesn't mint token for user
We have 3 ways how to solve it
1)Keep the function
2)Add the function to SequencerStaking.ts
3)Delete the function and add 2 line instead: minting and provideSequencerStaking

Reply:
then is fine, no worries, you can leave it here,
ty and sorry!